azenta_driver/README.md

```diff
@@ -0,0 +1,199 @@
+correctly to detect a particular seal type.
+<p>&nbsp;</p>
+
+7.     peeler.tape_remaining()
+
+The XPeel calculates the amount of tape remaining on the supply spool by comparing the travel of the
+conveyor to the rotation of both the supply and take-up spools. After a completed ‘XPeel’ operation, a
+‘tapeleft’ command can be issued to verify the amount of tape left as well as the space remaining on
+the take-up spool. These calculations are not perfectly accurate but they will provide sufficient
+warning to avoid running out of tape unexpectedly.
+<p>&nbsp;</p>
+
+8.     peeler.plate_check(y or n?)
+
+This command sets the plate check parameter to yes or no.
+<p>&nbsp;</p>
+
+9.     peeler.sensor_threshold()
+
+This command reports the threshold value of the seal detected sensor.
+<p>&nbsp;</p>
+
+10.     peeler.sensor_threshold_higher(Threshold Value)
+
+Setting the seal detected threshold value for the seal present if higher than threshold.
+
+The Threshold Value refers to the 3 digit threshold value. This command sets the seal detected threshold so that during an XPeel, sensor readings higher than this value represents an un-removed seal and will
+result in an error 04, “seal not removed”, message.
+<p>&nbsp;</p>
+
+11.     peeler.sensor_threshold_lower(Threshold Value)
+
+Setting the seal detected threshold value for the seal present if lower than threshold.
+
+Where Threshold Value refers to the 3 digit threshold value. This command sets the seal detected threshold so that during an XPeel, sensor readings lower than this value represents an un-removed seal and will
+result in an error 04, “seal not removed”, message.
+<p>&nbsp;</p>
+
+12.     peeler.conveyor_out()
+
+This command causes the conveyor to move out towards the user 7mm each time the command is called. When the conveyor reaches it’s fully extended home position no movement
+occurs.
+<p>&nbsp;</p>
+
+13.     peeler.conveyor_in()
+
+This command causes the conveyor to move in to the default ‘begin peel’ position.
+<p>&nbsp;</p>
+
+14.     peeler.elevator_down()
+
+This command causes the elevator to move down until it is stopped by a plate or reaches its lower limit. The plate check option is disabled for this command.
+<p>&nbsp;</p>
+
+15.     peeler.elevator_up()
+
+This command causes the elevator to move up approximately 1.5mm each time the
+command is called until the elevator reaches it full up home position.
+<p>&nbsp;</p>
+
+16.     peeler.move_spool()
+
+This command causes the spool to advance approximately 10mm of tape.
+
+<p>&nbsp;</p>
+
+### High Level Concepts
+1. Peeler information is initialized
+2. When a command is called a corresponding string is sent to the peeler.
+3. The Peeler interprets this string and ouputs data/performs an action accordingly.
+4. Python interprets output data from Peeler and displays messages while decoding error messages.
+		
+<p>&nbsp;</p>
+
+### Python Dependencies
```

Comment:
Usually no need for this. Pip takes care of it unless it is a "special" dependency